src/components/Calculator.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Output from './Output';
+import ButtonsBox from './ButtonsBox';
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
+      <div className="calculator">
+        <Output />
+        <ButtonsBox />
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] Kindly make this and other components to be functional components just as the requirements say.

![comp](https://user-images.githubusercontent.com/53124548/219616309-d1213ac0-8266-43fa-98c0-ffef60ed6883.png)